src/status_im/utils/utils.cljs

```diff
@@ -20,11 +20,11 @@
    (.alert (.-Alert rn-dependencies/react-native)
            title
            content
-           ; Styles are only relevant on iOS. On Android first button is 'neutral' and second is 'positive'
+           ;; Styles are only relevant on iOS. On Android first button is 'neutral' and second is 'positive'
```

Comment:
There is a chapter on comments in the informal clojure style guide https://github.com/bbatsov/clojure-style-guide#comments